### PR TITLE
Update elcinema.com.config.js

### PR DESCRIPTION
**Pull request by ksa-live**

### CODE REVIEW

File: sites/elcinema.com/elcinema.com.config.js [hunk 1]
```diff
@@ -77,7 +77,7 @@ function parseStart(item, initDate) {
 
   time = `${initDate.format('DD/MM/YYYY')} ${time}`
 
-  return dayjs.tz(time, 'DD/MM/YYYY H:mm A', 'Africa/Cairo')
+  return dayjs.tz(time, 'DD/MM/YYYY H:mm A', 'Saudi Arabia/Riyadh')
```
Comment (freearhey) [inline sites/elcinema.com/elcinema.com.config.js:80]: As far as I know, there is no such time zone.

https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

**Review by freearhey — CHANGES_REQUESTED** [review]

File: sites/elcinema.com/elcinema.com.config.js (hunk 1 above)
Comment (mcastellaneta) [reply sites/elcinema.com/elcinema.com.config.js:80]: @ksa-live I think you meant [Asia/Riyadh](https://en.wikipedia.org/wiki/Asia/Riyadh)

Comment (ksa-live) [reply sites/elcinema.com/elcinema.com.config.js:80]: @mcastellaneta yes that is the time i need

**Review by freearhey — DISMISSED** [review]

**Comment by mcastellaneta** [comment]
@ksa-live Could you update the times in the test file ? https://github.com/iptv-org/epg/blob/2e58a9a8ebd8dd3878bd8f2fba1bc3d16c01632b/sites/elcinema.com/elcinema.com.test.js#L32-L33 and https://github.com/iptv-org/epg/blob/2e58a9a8ebd8dd3878bd8f2fba1bc3d16c01632b/sites/elcinema.com/elcinema.com.test.js#L45-L46

**Review by freearhey — APPROVED** [review]

**Pull request merged**